release_notes_generator/generator.py

```diff
@@ -35,6 +35,7 @@
 logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
 
 
+# pylint: disable=too-few-public-methods
```

Comment:
I am not sure, if this has to be a class

Reply:
Yes, this represents an object of the generator.
Removed a Pylint disable and adding property methods for the class.